src/scaladoc/scala/tools/nsc/doc/base/CommentFactoryBase.scala

```diff
@@ -254,16 +254,16 @@ trait CommentFactoryBase { this: MemberLookupBase =>
         else lastTagKey match {
           case Some(key) =>
             val value =
-              ((tags get key): @unchecked) match {
+              (tags get key: @unchecked) match {
```

Comment:
can someone who knows the spec better confirm that `@unchecked` here matches `tags get key` and not just `key`?

Reply:
```
scala> import annotation.unchecked
import annotation.unchecked

scala> 1 + 2: @unchecked // print
   ((1).+(2): @unchecked) // : Int(3) @unchecked
```

Add `// print` in the repl and hit <kbd>tab</kbd> to show a pretty-print of the syntax tree

Reply:
I see. I will check again what impact that has. From what I saw in the code, the unchecked annotation continued to make sense here without the parentheses.

Reply:
I checked the specific code part and from my side the critical compiler warning in the pattern matching on tags get key is correctly suppressed. From what I see, the @unchecked flag is correctly applied without the parenthesis.

Reply:
I think `tags.get(key): @unchecked` is a more common sytle nowaday.